lib/google_scraping/account/queries/user_token_query.ex

```diff
@@ -0,0 +1,33 @@
+defmodule GoogleScraping.Accounts.UserTokenQueries do
```

Comment:
It should be like this
```suggestion
defmodule GoogleScraping.Accounts.Queries.UserTokenQueries do
```